lib/global_helper.js

```diff
@@ -18,6 +22,26 @@ if (Meteor.isClient) {
 	Template.registerHelper('listing', function () {
 		return Listing.find({ _id: id });
 	});
+
+	Template.registerHelper('online', function () {
+    		var status = Meteor.users.find( { _id: id } ).fetch()[0].status.online;
+    		if (status == true) {
+    			var color = "#24ec3d";
+    			var innercolor = "#029402";
+    			var text = "Online";
+    		}
+    		else {
+    			var color = "#ff0000";
+    			var innercolor = "#5858FD";
```

Comment:
'innercolor' is already defined.<br>Mixed double and single quotes.<br>'innercolor' is defined but never used.